### DISCUSSION TITLE
Bun v1.0.25

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#8330] [Announcements]
To install Bun v1.0.25:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.25:
```bash
bun upgrade
```

## Thanks to 5 contributors!

- [@bjon](https;//github.com/bjon)
- [@dylan-conway](https;//github.com/dylan-conway)
- [@Hanaasagi](https;//github.com/Hanaasagi)
- [@Jarred-Sumner](https;//github.com/Jarred-Sumner)
- [@nektro](https;//github.com/nektro)

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.0.24...bun-v1.0.25

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.25'>Bun v1.0.25</a>.</em>

**Comment by Chew** [comment]
The links to the contributors don't work :(
